### DISCUSSION TITLE
underground facilities

### DISCUSSION THREAD

**Discussion by imakihi** [TerriaJS/terriajs#5119] [Q&A]
I would like to show underground structures like subway stations and tunnels with Terria V8. Is it possible to do that? I saw several Cesium.js examples. It would be nice if I can show my underground stuff with Terria.

https://sandcastle.cesium.com/?src=Globe%20Translucency.html

Can anyone direct me a right direction?

Thank you!

Hiroo

**Comment by meh9** [comment]
By using the Compare function (a.k.a. “splitter”), on the controls on the left you can then set Terrain to be rendered on the left or right, and then you can also enable Terrain Hides Underground Features:

(Not a great screenshot but it shows the ability to swipe away the terrain)

<img width="1920" alt="Screen Shot 2021-01-11 at 16 41 40" src="https://user-images.githubusercontent.com/5283003/104393609-ffcdcf00-5598-11eb-9ebe-65a942fc1c59.png">
